test/tests/interfacekernels/InterfaceSorption/tests

```diff
@@ -121,10 +116,243 @@
+    mesh_mode = REPLICATED
+    recover = false
+  []
+  [Henry_transient_non_ad_penalty_scaling]
+    requirement = 'The system shall have the capability to enforce interfacial conditions based on the Henry law in isothermal conditions using a penalty-enforced flux balance during transient with unit scaling on both variables.'
+    type = 'Exodiff'
+    input = 'interface_sorption_transient.i'
+    cli_args = 'n_sorption=1
+                unit_scale=4
+                unit_scale_neighbor=6
+                InterfaceKernels/interface/use_flux_penalty=true
+                InterfaceKernels/interface/flux_penalty=1e1
+                Outputs/file_base=interface_sorption_transient_Henry_non_ad_penalty_scaling_out'
+    exodiff = 'interface_sorption_transient_Henry_non_ad_penalty_scaling_out.e'
+    # compared to Henry_transient_non_ad_scaling because it is supposed to be equivalent,
+    # but the residuals are expected different with the penalty so the relative error tolerance is increased here
+    rel_err = 5.4e-2
+    mesh_mode = REPLICATED
+    recover = false
+  []
+  [Henry_transient_ad_penalty_scaling]
+    requirement = 'The system shall have the capability to enforce interfacial conditions based on the Henry law in isothermal conditions using automatic differentiation and a penalty-enforced flux balance during transient with unit scaling on both variables.'
+    type = 'Exodiff'
+    input = 'interface_sorption_transient.i'
+    cli_args = 'n_sorption=1
+                unit_scale=4
+                unit_scale_neighbor=6
+                Kernels/u1_time_derivative/type=ADTimeDerivative
+                Kernels/u2_time_derivative/type=ADTimeDerivative
+                Kernels/temperature/type=ADTimeDerivative
+                Kernels/u1_diffusion/type=ADMatDiffusion
+                Kernels/u2_diffusion/type=ADMatDiffusion
+                Kernels/temperature/type=ADHeatConduction
+                InterfaceKernels/interface/type=ADInterfaceSorption
+                Materials/properties_1/type=ADGenericConstantMaterial
+                Materials/properties_1/type=ADGenericConstantMaterial
+                Materials/properties_2/type=ADGenericConstantMaterial
+                Postprocessors/flux_inner/type=ADSideDiffusiveFluxIntegral
+                Postprocessors/flux_outer/type=ADSideDiffusiveFluxIntegral
+                InterfaceKernels/interface/use_flux_penalty=true
+                InterfaceKernels/interface/flux_penalty=1e1
+                Outputs/file_base=interface_sorption_transient_Henry_ad_penalty_scaling_out
+                Executioner/nl_abs_tol=1e-12'
+    exodiff = 'interface_sorption_transient_Henry_ad_penalty_scaling_out.e'
+    # compared to Henry_transient_non_ad_scaling because it is supposed to be equivalent,
+    # but the residuals are expected different with the penalty so the relative error tolerance is increased here
+    rel_err = 5.4e-2
+    mesh_mode = REPLICATED
+    recover = false
```

Comment:
Note new issue number.

Reply:
Do you want a comment on the same line with the issue number?

Reply:
Yes - something like `See #196`

Reply:
Done